DESCRIPTION

```diff
@@ -0,0 +1,23 @@
+Package: IOHexperimenter
+Type: Package
+Title: Benchmarking Part of the 'IOHprofiler'
+Version: 0.0.0.9000
+Author: Furong Ye [cre, aut], Hao Wang [aut], Diederick Vermetten [aut], Carola Doerr [aut], Thomas Bäck [aut]
+Maintainer: Furong Ye <f.ye@liacs.leidenuniv.nl>
```

Comment:
The same thing here: are two maintainer allowed? I think it is better to set you or me as the maintainer

Reply:
For the maintainer field, only one person is allowed. Should I put you or me there?

Reply:
I think putting you would be good :)